File: src/prog_langs/php/update.php
<?php
	$db = [
		'host' => 'localhost',
		'name' => 'guest',
		'pass' => 'DK4XMDVATL2N8448CMF9',
		'db'   => 'php'
	];
	// dbに接続
	$mysqli = new mysqli("$db[host]","$db[name]","$db[pass]","$db[db]");
	// 文字コードをセット
	mysqli_set_charset($mysqli, 'utf8');

	$stmt = $mysqli->prepare('UPDATE toy SET name = ? WHERE id = ?');// toyにbindしたデータでupdate
	$stmt->bind_param('si', $name, $id);// この値をbindする、siはそれぞれの型を指定、str,int

	$name = 'robot';// bindする実際の値
	$id = 1;// bindする実際の値

	$stmt->execute();// ここで実行
	$mysqli->close();// dbを閉じる

File: src/prog_langs/python/update.py
import MySQLdb

# 接続するための情報
conn = MySQLdb.connect(
	user   = 'guest',
	passwd = 'DK4XMDVATL2N8448CMF9',
	host   = 'localhost',
	db     = 'python'
)

# カーソルを指定、python独自らしい
cur = conn.cursor()
# テーブルtoyにある指定のidの、nameカラムをupdateする文
sql = "UPDATE toy SET name = %s WHERE id = %s"
# 実行する際に、id:1、name:robotを指定
cur.execute(sql, ('robot', 1))
# 実行を保存
conn.commit()
# カーソルを終了
cur.close
# db接続を終了
conn.close